src/app/profile/info-form/info-form.component.html

```diff
@@ -1,55 +1,40 @@
+<br>
 <form class="example-form" [formGroup]="infoForm">
   <div class="grid-container">
     <div class="grid-item">
       <mat-form-field class="medium-field" appearance="outline" color="accent">
         <mat-label>First Name</mat-label>
-        <input matInput type="text" name="email" formControlName="email" />
+        <input matInput type="text" name="firstName" formControlName="firstName" value="{{profile?.name}}"/>
```

Comment:
Remove the value fields from every input.
```suggestion
        <input matInput type="text" name="firstName" formControlName="firstName"/>
```